locales/puppet.pot

```diff
@@ -6725,6 +6725,14 @@ msgstr ""
 msgid "Don't know how to install '%{source}'"
 msgstr ""
 
+#: ../lib/puppet/provider/package/xbps.rb:36
+msgid "Failed to match line '%{line}"
+msgstr ""
+
+#: ../lib/puppet/provider/package/xbps.rb:43
+msgid "Error getting installed packages"
+msgstr ""
+
```

Comment:
You can undo this part as this file is automatically generated from source.

Reply:
I'm not sure what this means, is it that it'll be generated automatically by CI/Actions in the future?

Reply:
Correct, the `puppet.pot` file shouldn't be modified in this PR. The file is autogenerated in our CI pipelines.